pallets/beefy-light-client/src/lib.rs

```diff
@@ -217,7 +233,7 @@ pub mod pallet {
             Ok(().into())
         }
 
-        #[pallet::weight(0)]
+        #[pallet::weight(<T as Config>::WeightInfo::submit_signature_commitment())]
```

Comment:
Looking at your benchmarks I think you need to rework the weight of submit_signature_commitment()